DQM/Integration/python/clients/l1tstage1emulator_dqm_sourceclient-live_cfg.py

```diff
@@ -219,8 +227,26 @@
     process.scalersRawToDigi.scalersInputTag = cms.InputTag("rawDataRepacker")
     process.siPixelDigis.InputLabel = cms.InputTag("rawDataRepacker")
     process.siStripDigis.ProductLabel = cms.InputTag("rawDataRepacker")
-
-
+    process.caloStage1Digis.InputLabel = cms.InputTag("rawDataRepacker")
+    process.load("L1Trigger.L1TCalorimeter.caloConfigStage1HI_cfi")
+    process.load('Configuration/StandardSequences/FrontierConditions_GlobalTag_condDBv2_cff')
+    process.GlobalTag.toGet = cms.VPSet(
+        cms.PSet(
+            record  = cms.string("L1TCaloParamsRcd"),
+            tag     = cms.string("L1TCaloParamsStage1HI_CRAFT09_hlt"),
+            connect = cms.string("frontier://(proxyurl=http://localhost:3128)(serverurl=http://localhost:8000/FrontierOnProd)(serverurl=http://localhost:8000/FrontierOnProd)(retrieve-ziplevel=0)(failovertoserver=no)/CMS_CONDITIONS")
+            )
+        )
+else:
+    process.l1ExpertDataVsEmulatorStage1.remove (process.l1tHIonImp)
```

Comment:
I don't think this is working.
I get l1tHIonImp module running both in pp and in HeavyIonsRun2 configurations